build.py

```diff
@@ -109,16 +109,20 @@ def create_package_fs(build_root):
         create_dir(os.path.join(build_root, d))
         os.chmod(os.path.join(build_root, d), 0755)
 
-def package_scripts(build_root):
+def package_scripts(build_root, static=False):
```

Comment:
> Also why do static build not need the extra service scripts etc? Is it because they are intended for use in apline linux? If so a comment explaining that would be helpful.

That is correct. The only people who should be using the static builds are people who just need binaries, so Alpine users and Windows users are the first ones who come to mind. Do you think all tars and zips should be this way?

> The name static for this method is confusing maybe we should name this parameter config_only?

Agreed. I'll change it. I was originally thinking of 'static' packages (just plain zip, tar), but I can see how it's misleading and/or vague.